.github/json_matrices/supported-language-versions.json

```diff
@@ -0,0 +1,27 @@
+[
+    {
```

Comment:
I did it myself in your recommendation, but why we run just one when there's less option? I think it's not the amount available, it's about testing the supported range with the highest and lowest, so if we support just two version this is our range, and we test all of it.

Reply:
Same for go and C# altho maybe C# is not under current development, so there's no need, we just want to keep it kind of under control.